samples/samples-powershell/AddProductWithIdentityColumn/run.ps1

```diff
@@ -7,7 +7,7 @@ param($Request, $TriggerMetadata)
 Write-Host "PowerShell function with SQL Output Binding processed a request."
 
 # Update req_query with the query of the request
```

Comment:
These comments aren't useful, let's just remove them. Instead it would be great if you could replace it with an explanation of why you're using ordered.

Reply:
> Please also add a note to the Python docs about when and why users should use ordered in their own functions.

Did you mean PowerShell docs?

Reply:
Yes 😁

Reply:
Sounds good, are there the docs you are referring to [here](https://github.com/Azure/azure-functions-sql-extension/blob/main/docs/SetupGuide_PowerShell.md#output-binding) to confirm? 

I can remove the other comments in another PR and the linting issue.

Reply:
We've been putting stuff like that in https://github.com/Azure/azure-functions-sql-extension#known-issues

Although it might be better to have a language-specific known issues section for each language guide (that would then be linked to from the main README). But for now just keep it with the others and we can move them later if we decide that's useful.